src/components/slider/Slider.module.css

```
@@ -0,0 +1,262 @@
+.stepsSubContainer {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 98%;
+  height: 100%;
+}
+
+.stepMarker {
+  width: 4px;
+  height: 4px;
+  background: var(--surface-hover);
```

Stylelint (17.3.0)

[error] Unexpected duplicate "background" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)